doc/rst/technotes/initializers.rst

```diff
@@ -12,123 +12,289 @@ than the original methods known as constructors.
 A discussion of the current design and rationale is provided in
 `CHIP 10 <https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/chips/10.rst>`_.
 
-Release Chapel 1.15.0 provides a strong preview implementation of this
-new feature. This initial implementation is not of production quality
-and has not been heavily tested, but the basic features are working
-and it is believed that adventurous developers will have success using
-init methods in new code.
+Release Chapel 1.16.0 provides a strong preview implementation of this
+new feature.  Though some known bugs remain, the feature is rapidly
+approaching full support and developers should feel encouraged to
+write initializers where previously they had relied on constructors.
 
 It is anticipated that the implementation will continue to advance
 steadily after this release and that many aspects of the internal
 implementation of Chapel will transition from constructor methods to
 initializer methods during this release.
 
 
-Stability
----------
+The Initializer Method
+----------------------
 
-The functionality of an initializer method has four major aspects
+An initializer is a method on a class or record named "init".  It is invoked
+by the ``new`` operator, where the type name and initializer arguments are
+preceded with the ``new`` keyword.
 
-  - user-defined code that runs in phase 2
+If the program declares a type initializer method, it is a user-defined
+initializer.  If the program declares no initializers or constructors for a
+class or record, a compiler generated constructor for that type is created
```

Comment:
"a compiler generated constructor for that type" --> "a compiler generated initializer for that type"

Reply:
I'm describing the current state here, rather than the intended direction, so it should stay "constructor" for now.  Should I call out there that we intend to move to compiler generated initializers, rather than just having a reference to the section on them?